src/hooks/useKonfluxPublicInfo.ts

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useK8sWatchResource } from '../k8s';
+import { ConfigMapGroupVersionKind, ConfigMapModel } from '../models';
+import { KonfluxPublicInfo, KonfluxPublicInfoConfigMap } from '../types/konfluxPublicInfo';
```

Comment:
As suggested, the separate hook for konflux public info and related types has been created.